components/EligibilityPage/index.tsx

```diff
@@ -261,7 +261,7 @@ export const EligibilityPage: React.VFC = ({}) => {
                 label={field.config.label}
                 onChange={debounce(
                   (e) => handleOnChange(field, e.target.value),
-                  100
+                  500
```

Comment:
although it helps, still has issues when entering the income very slow, not sure why is this happening as older branches didn't have this issue

Reply:
hmm, I tested again and it seems to be working fine on initial input but when going back to the input, the bouncing back and forth comes back. Darn.. I'm not sure what caused this to start happening but I will investigate a bit more.

Reply:
Ah, I simply needed to move the setState for error visibility after React actually deals with changing the input state!